CMakeLists.txt

```diff
@@ -42,13 +38,15 @@ if(BUILD_DBoW2)
     set(LIB_SHARED "STATIC")
   endif(WIN32)
   add_library(${PROJECT_NAME} ${LIB_SHARED} ${SRCS})
-  include_directories(include/DBoW2/)
+  target_include_directories(${PROJECT_NAME} PUBLIC include/DBoW2/)
```

Comment:
Would this by chance fix the issue with expected location of the exported headers in downstream projects? https://github.com/MIT-SPARK/Kimera-VIO/pull/32#discussion_r390787620

Reply:
I think it will. I'm close to have this branch merged, so you will be able to test it.